src/condor_starter.V6.1/jic_shadow.cpp

```diff
@@ -1098,6 +1099,36 @@ JICShadow::notifyJobTermination( UserProc *user_proc )
 	return rval;
 }
 
+// send a command with a classad payload and receive a classad payload in reply.
+ClassAd * JICShadow::sendStartdCommmand(int cmd, ClassAd & payload)
```

Comment:
This isn't used anywhere and it's mis-spelled.

Reply:
fine with deleting it for now.

Reply:
it's a stub for ToddM to use if he needs it.